### DISCUSSION TITLE
Animasjon i Jøkul

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2923] [Annet]
Vi har litt diverse dokumentasjon og prinsipper på animasjon [i portalen](https://jokul.fremtind.no/profil/animasjon). Dette er egentlig ganske vagt og gir ikke veldig mye forklarende merverdi. I tillegg har vi et lite (og udokumentert) API i Jøkul, men det er minimalt _(se følgende filer: [motion](https://github.com/fremtind/jokul/blob/a9ef7e7b8b7b04d90cdc9300aced2e68ea341120/packages/core/mixins/_motion.scss), [easing](https://github.com/fremtind/jokul/blob/a9ef7e7b8b7b04d90cdc9300aced2e68ea341120/packages/core/functions/_easing.scss) og [timing](https://github.com/fremtind/jokul/blob/a9ef7e7b8b7b04d90cdc9300aced2e68ea341120/packages/core/functions/_timing.scss))_ og man vet ikke at det er der med mindre noen forteller deg om det.

## 🔮  Mål
Vi ønsker å implementere mer animasjon i løsningene våre. For å kunne gjøre dette effektivt burde Jøkul ha god dokumentasjon på temaet (med eksempler) som kan tas i bruk av teamene eller bidra til inspirasjon.

## 🎯 Oppgaver
Hva trenger vi for å nå dette målet? Hvilke aktiviteter må planlegges for å få dette på plass? Hvem vil være med på å forme dette? Meld interesse ved å kommentere på denne tråden, så samler vi en gruppe folk som kan jobbe med det fokusert.

**Comment by wkillerud** [comment]
Jeg er med 🙋

**Reply by onordbo** [reply]
Ditto 🚀
